snapdtool/cmdutil.go

```diff
@@ -107,6 +107,11 @@ func CommandFromSystemSnap(name string, cmdArgs ...string) (*exec.Cmd, error) {
 	}
 
 	cmdPath := filepath.Join(root, name)
+
+	if from == "snapd" {
```

Comment:
It is somewhat unclear what is going on here. Would you mind expanding the commit message? Also include a comment as to why snapd is excluded please.

Reply:
Maybe I should also add a comment in the code as well. I agree it is not obvious. It is just that all the hacks under are not needed anymore... unless we are talking about snapd tools from `core` snap instead of `snapd` snap. Will fix that.

Reply:
Added comments. And also more in the commit message.